doc/LETS_ENCRYPT_CONFIG.md

```diff
@@ -0,0 +1,49 @@
+Here are the steps to get Let's Encrypt SSL certificates on Chef server
+
+##Prerequisites
+
+1. Chef server must be reachable on port 80 or 443 on a public IP address
+2. A public fqdn configured and pointing to your Chef server IP address. For example: chef.example.com
+3. One of the [Let's Encrypt ACME Client](https://letsencrypt.org/docs/client-options) of your choice. 
+
+
+##Configuration
+The next steps can be performed with the [Certbot](https://certbot.eff.org/), recommended general purpose by Let's Encrypt  following steps 
+
+1. Follow the steps to get Certbot installed on the Chef server
+2. Execute certbot command to create the certificates, where -d is your Chef server public fqdn and -m your email address
+
+```
+$ certbot certonly --webroot -w /var/opt/opscode/nginx/html -d chef.example.com -m your@email.address
+```
+
+3. Edit /etc/opscode/cher-server.rb to add the SSL certificates 
+
+```
+api_fqdn 'chef.example.com'
+nginx['ssl_certificate']='/etc/letsencrypt/live/chef.example.com/fullchain.pem'
+nginx['ssl_certificate_key']='/etc/letsencrypt/live/chef.example.com/privkey.pem'
+nginx['ssl_ciphers'] = "HIGH:MEDIUM:!LOW:!kEDH:!aNULL:!ADH:!eNULL:!EXP:!SSLv2:!SEED:!CAMELLIA:!PSK"
+nginx['ssl_protocols'] = "TLSv1 TLSv1.1 TLSv1.2"`
+```
+
+4. Reconfigure the Chef server
+```
+chef-server-ctl reconfigure
+```
+
+5. At this point your Chef server is now using the newly created SSL certificates. You can browse the Chef server URL. For example: https://chef.example.com
+
+6. A Let's Encrypt certificate is only valid for 90 days, therefore it requires to be renewed frequently. 
+In this case, you can configure cron to run the certbot command below every day. By default, the certificate will only be renewed if it is valid for 30 days or less. 
+
+```
+certbot renew --renew-hook '/opt/opscode/init/nginx restart'
```

Comment:
Yes sure. I have always restarted the whole server when I needed, I didn't know I could restart just nginx with chef-server-ctl.